src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -11,11 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Are there any `img` folder anymore? We might be able to remove the `ItemGroup`

Reply:
@jorive #done